### PR TITLE
Allow user to submit request

### PR DESCRIPTION
**Pull request by richard-dds**
Also includes logic to hide certain parts of the form depending on the status of the request.

The summary step (4) is only shown after the request has been fully filled-out. The financial verification step is shown only after approval.

Had some merge conflicts on a couple templates during my rebase, so it may be worth doing a quick UI sweep on the requests form.

### CODE REVIEW

File: atst/handlers/request_new.py [hunk 1]
```diff
@@ -109,16 +83,120 @@ def get_request(self, request_id):
...
+                "title": "Review & Submit",
+                "section": "review_submit",
+                "form": ReviewForm,
+                "show": self.request and self.request["status"] == "pending_submission",
+            },
+            {
+                "title": "Financial Verification",
+                "section": "financial_verification",
+                "form": FinancialForm,
+                "show": self.request and self.request["status"] == "approved",
```
[the remark this answers is not in the document]
Comment (richard-dds) [reply atst/handlers/request_new.py]: Good catch! I'll fix this.

[the remark this answers is not in the document]
Comment (richard-dds) [reply atst/handlers/request_new.py]: @luisgov Do you have any idea what should happen here? Should the "review and submit" section still be available after the request is already approved?

[the remark this answers is not in the document]
Comment (luisgov) [reply atst/handlers/request_new.py]: @richard-dds _Review & Submit_ should always be available. However, a user should not be able to submit until all fields are completed.

_Financial Verification_ should not be displayed (or be grayed out/disabled) until the request is approved.

[the remark this answers is not in the document]
Comment (richard-dds) [reply atst/handlers/request_new.py]: @patricksmithdds Fixed.

**Pull request merged**